rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandler.groovy

```diff
@@ -161,6 +162,9 @@ public class AWSBakeHandler extends CloudProviderBakeHandler {
     logsContent.eachLine { String line ->
       if (line =~ IMAGE_NAME_TOKEN) {
         imageName = line.split(" ").last()
+      } else if (line =~ UNECRYPTED_IMAGE_NAME_TOKEN) {
```

Comment:
Maybe add a test to the unit tests here: https://github.com/spinnaker/rosco/blob/master/rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/aws/AWSBakeHandlerSpec.groovy